src/js/view/view.js

```diff
@@ -790,6 +793,10 @@ function View(_api, _model) {
             _controls.destroyInstream(_model);
         }
 
+        if (floatingPlayer === _playerElement) {
+            _this.floatingUI = new FloatingDragUI(_wrapperElement);
```

Comment:
Why don't you just manage dragging via  `_this.floatingUI.disable()` and `_this.floatingUI.enable()`? That way this logic is encapsulated in `FloatingDragUI ` instead of here in the view, and you don't have to (duplicate code) create new instances.